prow/cluster/jobs/istio/istio.io/istio.istio.io.master.yaml

```diff
@@ -0,0 +1,16 @@
+presubmits:
+
+  istio/istio.io:
+
+    - name: presubmit-master
+      path_alias: istio.io/istio.io
+      decorate: true
+      always_run: true
```

Comment:
I think you can add `branches: (release-1.2|master|release-1.3)` and consolidate, or if you want them isolate you need to add `branches: master` or something similar. Not 100% sure on this though

Reply:
@cjwagner Can you clarify  what I should be using here?

And what does it mean when  there is no branches spec at all, such as is the case in the  yaml for the pkg and community repos? Does that imply "for all branches"? if so, then should  I rename the yaml files to remove "master" from them, or is the name of the files somehow significant?

Reply:
If there are no `branches` or `skip_branches` then the job runs on all branches. So as written, this PR adds 3 identical jobs with different names that all run on all branches.
Unless you need the job to run with a different name on different branches you can just keep one version of the job and let it run against all branches (or specify `branches: [master, release-1.2, release-1.3]` to just run on these branches).

The name of the file is not significant except that it needs to be unique from the other file names since the file name is used as a configmap key.